components/PanelsList.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import { View, FlatList, StyleSheet } from "react-native";
+import NetInfo from "@react-native-community/netinfo";
+import { Snackbar } from "react-native-paper";
+
+import ListItem from "../components/ListItem";
+import { SERVER_API_URL } from "../consts/db";
+
+export default class PanelsList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      data: [],
+      snackbarMessage: "",
+    };
+  }
+
+  handleRenderItem = ({ item }) => <ListItem item={item} />;
```

Comment:
гадаю префікс `handle`- не дуже коректний для render методу і його варто було б зробити звичайною функцією тоді
```suggestion
 renderListItem({ item }) {
   return <ListItem item={item} />;
 }
```